templates/watcher/config/00-default.conf

```diff
@@ -59,3 +59,13 @@ datasources = ceilometer
 [cache]
 memcached_servers = {{ .MemcachedServers }}
 {{ end }}
+
+{{ if (index . "PrometheusHost") }}
+[prometheus_client]
+host = {{ .PrometheusHost }}
+port = {{ .PrometheusPort }}
+fqdn_label = fqdn
+{{ if (index . "PrometheusTLS") }}
+cafile = {{ .PrometheusCaCert }}
+{{ end }}
+{{ end }}
```

Comment:
if we are adding these here then may as well also include cerfile and keyfile?

Reply:
cafile is to validate certificate coming from a SSL enabled service.
cerfile and keyfile are for client side authentication using an SSL certificate. I think we don't support that.

Reply:
i was assuming we would make all the prometheus_client vars added in https://review.opendev.org/c/openstack/watcher/+/934423 available here. but per the other discussion about the host/port and not supporting the external prometheus deployment i realise we are going to be more opinionated in the operator. marking resolved thanks